src/frameworks/index.ts

```diff
@@ -109,8 +109,15 @@ const SupportLevelWarnings = {
 export const FIREBASE_FRAMEWORKS_VERSION = "^0.6.0";
 export const FIREBASE_FUNCTIONS_VERSION = "^3.23.0";
 export const FIREBASE_ADMIN_VERSION = "^11.0.1";
-export const DEFAULT_REGION = "us-central1";
 export const NODE_VERSION = parseInt(process.versions.node, 10).toString();
+export const DEFAULT_REGION = "us-central1";
+export const ALLOWED_SSR_REGIONS = [
```

Comment:
Hi, is there a motive behind limiting the regions? For example, we currently use `europe-west2` (London) but it seems this configuration would be disallowed?

Reply:
That's because cloud Functions V2 are only available in those regions.

Reply:
@tamerxkilinc This page https://cloud.google.com/functions/docs/locations?hl=en#tier_2_pricing shows that Cloud Functions v2 are currently supported in many more regions. Does that mean we can already use them or are there other limiting factors?

Reply:
Should be possible to use the new regions now